roadmap.md

```diff
@@ -0,0 +1,66 @@
+- Semantic conventions for metadata, which define how resources and other components are represented. This is applied consistently across all signal types, which allows traces, metrics, and other signals to be processed consistently and correlated.
+- SDKs for twelve languages that allow developers to capture telemetry from their services and to create their own custom telemetry.
+- Automatic instrumentation agents for __ languages, which allow anyone to capture telemetry from their services without making code changes or redeploying.
+- Instrumentation packages for thousands of pieces of software, including operating systems, container runtimes, databases, language runtimes, and libraries, all of which allow the Collector (for infrastructure and third-party applications), and SDKs and automatic instrumentation agents (for custom applications) to capture signals, capture metadata, and propagate context automatically.
+- A protocol, OTLP, for transmitting telemetry and metadata between OpenTelemetry components and to backends for processing.
+
+# Major Priorities
+All of the following workstreams are major areas of investment for OpenTelemetry: all of them have large groups of people already focused on them. Their priorities primarily reflect the ordering of when we expect them to be released.
+
+## P0: Continued Investment in OpenTelemetry Artifacts
+The project’s top priority will always be ensuring that the capabilities and robustness of our existing artifacts (the Collector, SDKs, language agents, etc.) remain excellent. This work takes place in each language SIG and in the Collector SIG, and includes continual improvement to these components, making them even easier to use, providing and integrating more instrumentation packages, working with the suppliers of telemetry sources to use the OpenTelemetry APIs natively, and more.
+
+## P1: Logs
+OpenTelemety established a logging SIG in mid-2020, with two goals:
+
+1. Providing a performant path for capturing logs from existing sources (typically text files on disk), where all captured logs have OpenTelemetry’s metadata consistently applied to them.
+2. Providing a new, strongly-typed and extremely high-performance logging path for new applications, which allows logs to be authored and transmitted without being parsed from text, and which enforces the consistency of all metadata.
+
+Much progress has been made on the first item, particularly through the donation of the Stanza logging agent into the OpenTelemetry Collector, and by the investments that the project has made in defining and stabilizing a data model and OTLP format for logs. However much still remains before we can declare our logging work generally available:
+
+- We must extend the Collector’s support for existing log sources to satisfy the requirements of more users.
+- Elastic donated the Elastic Common Schema to OpenTelemetry’s semantic conventions. We must fully integrate this.
```

Comment:
Can we get a positive confirmation from Elastic folks that they are interested in this? I remember that the original author of the PR closed it, as nobody from Elastic seemed interested in picking up the work after he left.

https://github.com/open-telemetry/oteps/pull/199#issuecomment-1331097321

Reply:
The proposal was [last updated](https://github.com/open-telemetry/oteps/issues/197) in March by the same person who ended up closing the PR.

If Elastic is still interested in moving forward with this, it shouldn't be hard to get a positive confirmation from them.

Reply:
Hi @jpkrohling,
we are still interested!
I opened a [new PR for the proposal](https://github.com/open-telemetry/oteps/pull/222) and looking forward to moving forward on this!

Reply:
Nice, I'm happy to see this moving forward!